pkgs/misc/themes/adapta/default.nix

```diff
@@ -33,11 +82,18 @@ stdenv.mkDerivation rec {
 
   postPatch = "patchShebangs .";
 
-  configureFlags = [
-    "--disable-gtk_legacy"
-    "--disable-gtk_next"
-    "--disable-unity"
-  ];
+  configureFlags = [ "--enable-parallel" ]
+    ++ (disableFlag "gnome" enableGnome)
```

Comment:
`disableFlag` returns a string, so why the `++`?

Reply:
It returns an array (using lib.optional).

But I'd rather use `lib.enableFeature` like you suggested.